Source/Core/loadImageFromTypedArray.js

```diff
@@ -21,6 +31,21 @@ define([
             type : format
         });
 
+        // Avoid an extra fetch by just calling createImageBitmap here directly on the blob
+        // instead of sending it to Resource as a blob URL.
+        if (FeatureDetection.supportsCreateImageBitmap()) {
+            return Resource.supportsImageBitmapOptions()
+                .then(function(supportsBitmapOptions) {
+                    if (supportsBitmapOptions) {
+                        return when(createImageBitmap(blob, {
+                            imageOrientation: flipY ? 'flipY' : 'none'
+                        }));
+                    }
+
+                    return when(createImageBitmap(blob));
```

Comment:
No need for these `when` wrappers.

Reply:
I believe this is necessary. See https://github.com/AnalyticalGraphicsInc/cesium/pull/7579#discussion_r258934322